config/environments/test.rb

```diff
@@ -21,7 +23,7 @@
   config.action_controller.perform_caching = false
 
   # Raise exceptions instead of rendering exception templates.
-  config.action_dispatch.show_exceptions = false
+  config.action_dispatch.show_exceptions = true
```

Comment:
We need to reverse this. We're doing this because bootstrap is not loading the icons correctly.